earth_enterprise/rpms/buildSrc/src/main/groovy/GeeDeb.groovy

```diff
@@ -82,4 +82,19 @@ class GeeDeb extends com.netflix.gradle.plugins.deb.Deb {
 
         findRequires(packageInputFiles).each { requires(it) }
     }
-}
\ No newline at end of file
+
```

Comment:
Please add a copyright header (copied from build.gradle should be fine)

Reply:
Sure thing. Just the part from the first line down to the line "limitations under the License"?

Reply:
Yep, you got it!

Reply:
Good, because I went ahead and committed it earlier ;)